docs/source/templates/class_api.rst

```diff
@@ -0,0 +1,197 @@
+..
+  Use the following convention for headings:
+
+    # with overline, for parts
+
+    * with overline, for chapters
```

Comment:
This is just copied and pasted from the sphinx docs, overline is because you can have headings like this:

```rst
#######
Heading
#######
```

and

```rst
*******
Heading
*******
```

But the others shouldn't have overlines

Reply:
ah ok!